src/qtism/data/content/interactions/ExtendedTextInteraction.php

```diff
@@ -162,6 +162,9 @@ class ExtendedTextInteraction extends BlockInteraction implements StringInteract
      */
     private $format = TextFormat::PLAIN;
 
+    // This option disable also validation for patternMask
+    private $isDisabledMaxWordValidation = false;
```

Comment:
could you please clarify a bit where  and how the property is used? can't find it in this PR and related

Reply:
sorry, yes it is not used